src/Schema.js

```diff
@@ -591,14 +582,14 @@ class Schema {
     if (!this.perms[className] || !this.perms[className][operation]) {
       return Promise.resolve();
     }
-    var perms = this.perms[className][operation];
+    let perms = this.perms[className][operation];
     // Handle the public scenario quickly
     if (perms['*']) {
       return Promise.resolve();
     }
     // Check permissions against the aclGroup provided (array of userId/roles)
-    var found = false;
-    for (var i = 0; i < aclGroup.length && !found; i++) {
+    let found = false;
+    for (let i = 0; i < aclGroup.length && !found; i++) {
       if (perms[aclGroup[i]]) {
         found = true;
```

Comment:
Or even better: use `Array.prototype.some`. But I'd rather do that in a different diff.